modules/installation-initializing-manual.adoc

```diff
@@ -37,6 +49,9 @@ ifdef::azure-gov[]
 When installing {product-title} on Microsoft Azure into a government region, you
 must manually generate your installation configuration file.
 endif::azure-gov[]
+ifdef::aws-private,azure-private,gcp-private[]
+For installations of a private {product-title} cluster that are only accessible from an internal network and are not visible to the internet, you must manually generate your installation configuration file.
```

Comment:
s/internet/Internet/

per IBMSG, tho looks like usage is flip-flopping in the wider, built assemblies.

Reply:
Yes, I've seen it both ways. Based on a recent peer review, I've been lower-casing. I did not know this was stated in the IBMSG :sweat_smile: . I will upper-case from now on! Great feedback; thanks!